package.json

```diff
@@ -42,10 +42,13 @@
 		"@web/test-runner": "^0.18.3",
 		"chai": "^5.1.1",
 		"htm": "^3.1.1",
-		"preact": "10.15.1",
-		"preact-render-to-string": "^6.4.0",
+		"preact": "^10.23.2",
+		"preact-render-to-string": "^6.5.9",
 		"sinon": "^18.0.0",
 		"sinon-chai": "^4.0.0",
 		"uvu": "^0.5.6"
+	},
+	"resolutions": {
+		"@web/dev-server-core": "0.7.1"
```

Comment:
Bug with wtr -- failed sinon expectations are picked up as test failures (outside of manual mode, anyways). Downgrading for now helps.

See https://github.com/modernweb-dev/web/issues/2772